rclpy/test/test_logging.py

```diff
@@ -106,6 +106,14 @@ def test_log_throttle(self):
             ))
             time.sleep(0.4)
         self.assertEqual(message_was_logged, [True, False, False, True, False])
+        self.assertEqual(
+            message_was_logged, [
+                True,  # t=0, not throttled
+                False,  # t=0.4, throttled
+                False,  # t=0.8, throttled
+                True,  # t=1.2, not throttled
+                False  # t=1.6, throttled
+            ])
```

Comment:
Is this check now just duplicated?

Reply:
the other uses skip first as well

Reply:
I think @wjwwood meants that it looks like the line above can be removed

Reply:
ah! of course. thanks: 085e860302235ee394a7776576600d9e3f607433